pyaerocom/web/helpers_evaluation_iface.py

```diff
@@ -1417,6 +1421,13 @@ def compute_json_files_from_colocateddata(coldata, obs_name,
             _write_stationdata_json(ts_data, out_dirs['ts'])
 
     if coldata.ts_type == 'hourly':
+        #writing hourly heatmap
+        hm_fname_hourly = HEATMAP_FILENAME_EVAL_IFACE_HOURLY
+        hm_file_hourly = os.path.join(out_dirs['hm'], hm_fname_hourly)
+        hm_data = hm_all['hourly']
+        _add_entry_heatmap_json(hm_file_hourly, hm_data, web_iface_name, obs_var,
```

Comment:
would be great if you could add some tests for the diurnal processing!

Reply:
I will try. good opportunity to start writing tests